### ISSUE TITLE
Cannot push my repo

### ISSUE THREAD

**Issue by urielhdz** [nodester/nodester#399]
I'm sorry about my english buy I'm from Mexico and I want to start to deploy Node.js apps by using Node

This is my current app http://chat_nodejs.nodester.com/

I think that until ther I'm good, now I want to push my files.

I followed the next steps.

nodester app init chat_nodejs
git remote set-url nodester git@nodester.com:/node/git/dxv/12038-d75a4ecfaf0a7d17b6fc7f5f7f5eb872.git
git push nodester master

And then I got the follow sentences:
Nodester!
Enter passphrase for key... : I wrote my passphrase
error: src refspec master does not match any.
error: failed to push some refs to 'git@nodester.com:/node/git/dxv/12038-d75a4ecfaf0a7d17b6fc7f5f7f5eb872.git'

And thats it, my app still under the prototype and I can't move forward with this.

I have to say that I'm not an expert on github

**Comment by alejandro** [comment]
Yeah! You should check your RSA key. We have documents for that _> [http://nodester.com/help](http://nodester.com/help)
